ci-operator/step-registry/openshift/upgrade/aws/single-node/openshift-upgrade-aws-single-node-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+workflow:
+  as: openshift-upgrade-aws-single-node
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-hosted-loki
+    - chain: ipi-conf-aws
+    - ref: single-node-conf-aws
+    - chain: ipi-install-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: gather-network
+    - chain: gather-core-dump
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
+      TEST_TYPE: upgrade
+      TEST_SUITE: ""
+      TEST_UPGRADE_SUITE: ""
+      TEST_UPGRADE_OPTIONS: ""
+      TEST_SKIPS: "disruption_tests"
+  documentation: |-
+    The Openshift E2E AWS single node workflow executes the upgrade end-to-end test suite on AWS with a default single-node cluster configuration and Loki as log collector, exercising an upgrade during the process.
```

Comment:
is loki is something specific here? Just so i understand, or this is the standard flow

Reply:
Copied from regular upgrade workflow

Reply:
This step would setup promtail on the cluster, sending container and journald logs to the central repo, which can be queried by Grafana.

This is a standard practice for upgrade workflows, as intermediate containers can be lost during reboot (or the node may never come back)